homatransport/src/application/WorkloadSynthesizer.cc

```diff
@@ -30,6 +31,8 @@ simsignal_t WorkloadSynthesizer::msgE2EDelaySignal =
     registerSignal("msgE2EDelay");
 simsignal_t WorkloadSynthesizer::mesgStatsSignal = registerSignal("mesgStats");
 
+std::ofstream outputFile;
+
 WorkloadSynthesizer::WorkloadSynthesizer()
 {
     msgSizeGenerator = NULL;
```

Comment:
Would it be possible to move to Google naming style for variables and functions? I sent the style guide to you in chat

Reply:
Like we discussed, for this project using camelCase for this project.